venues/schema.py

```diff
@@ -82,7 +82,10 @@ def resolve_occurrences(self, info, **kwargs):
         )
 
     def resolve_translations(self, info, **kwargs):
-        return self.translations.order_by("language_code")
+        try:
+            return self.prefetched_translations
+        except AttributeError:
+            return self.translations.order_by("language_code")
```

Comment:
The order_by ascending/descending order differs between the prefetched_translations and this version (i.e. order_by("-language_code") vs. order_by("language_code")).

Reply:
Well spotted 👀. Changed.